browser/src/Services/Explorer/ExplorerView.tsx

```diff
@@ -49,36 +78,71 @@ export class NodeView extends React.PureComponent<INodeViewProps, {}> {
     }
 
     public getElement(): JSX.Element {
-        const node = this.props.node
+        const { node } = this.props
 
         switch (node.type) {
             case "file":
                 return (
-                    <SidebarItemView
-                        text={node.name}
-                        isFocused={this.props.isSelected}
-                        isContainer={false}
-                        indentationLevel={node.indentationLevel}
-                        icon={<FileIcon fileName={node.name} isLarge={true} />}
+                    <DragAndDrop
```

Comment:
This seems very intuitive! 👍

Reply:
😄The library is really quite nice but I fumbled with it a bit initially spent a week making no progress because I didn't realise how the items for dragging and dropping *ought* to be labelled so decided to separate these out into render props that theoretically someone else could use that seemed a little bit more straightforward.